build-tools/libzip/libzip.targets

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project DefaultTargets="Build" ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <_SourceTopDir>..\..</_SourceTopDir>
+  </PropertyGroup>
+  <UsingTask AssemblyFile="$(_SourceTopDir)\bin\Build$(Configuration)\Xamarin.Android.Tools.BootstrapTasks.dll" TaskName="Xamarin.Android.Tools.BootstrapTasks.GetNugetPackageBasePath" />
+  <Import Project="libzip.props" />
+  <Import Project="libzip.projitems" />
+  <Target Name="_SetCMakeListsTxtTimeToLastCommitTimestamp">
+    <Exec
+        Command="touch -m -t `git log -1 --format=%25cd --date=format-local:%25Y%25m%25d%25H%25M.%25S` CMakeLists.txt"
+        WorkingDirectory="$(LibZipSourceFullPath)"
+    />
+  </Target>
+  <Target Name="_Configure" DependsOnTargets="_SetCMakeListsTxtTimeToLastCommitTimestamp">
+    <MakeDir Directories="@(_LibZipTarget->'$(IntermediateOutputPath)\%(Identity)')" /> 
+    <Exec
+        Command="%(_LibZipTarget.CMake) %(_LibZipTarget.CMakeFlags) $(LibZipSourceFullPath)"
+        WorkingDirectory="@(_LibZipTarget->'$(IntermediateOutputPath)\%(Identity)')"
+    />
+  </Target>
+  <Target Name="_Make"
+        Inputs="$(LibZipSourceFullPath\CMakeLists.txt"
```

Comment:
Similarly, the `Inputs` here should actually be one of the Outputs from `_Configure`.